### ISSUE TITLE
Preview not available

### ISSUE THREAD

**Issue by TitusEfferian** [mirukusheki/nhentai-generator#5]
preview is not available, is this issue or the feature is not done yet?

**Comment by mirukusheki** [comment]
The feature is already done. But had been blocked by CORS policy. 
This time we are trying to fix it, or maybe remove those feature because it require another server to fetch data from another site.

**Comment by py7hon** [comment]
> The feature is already done. But had been blocked by CORS policy.
> This time we are trying to fix it, or maybe remove those feature because it require another server to fetch data from another site.

Use [This](https://nhproxy.glitch.me) for bypass the CORS
